src/components/Tool/FixCommitmentInfo/Contact.tsx

```diff
@@ -79,225 +123,455 @@ const useStyles = makeStyles()(() => ({
-              >
-                {"Don't Change"}
-              </Button>
-            </Box>
-            <Box>
-              <IconButton
-                data-testid="goToContactsButton"
-                onClick={() =>
-                  push({
-                    pathname: `/accountLists/[accountListId]/contacts/[contactId]`,
-                    query: { accountListId, contactId: id },
-                  })
-                }
-              >
-                <SearchIcon />
-              </IconButton>
-              <IconButton
-                data-testid="hideButton"
-                onClick={() => hideFunction(id)}
-              >
-                <VisibilityOffIcon />
-              </IconButton>
-            </Box>
-          </Box>
-        </Grid>
-      </Grid>
+      <Formik
+        initialValues={{
+          statusValue: statusValue,
+          pledgeCurrency: amountCurrency,
+          pledgeAmount: amount,
+          pledgeFrequency: frequencyValue,
+        }}
+        validationSchema={appealFormSchema}
+        onSubmit={async (values) => {
+          await onSubmit(values);
+        }}
+      >
+        {({
+          values: {
+            statusValue,
+            pledgeCurrency,
+            pledgeAmount,
+            pledgeFrequency,
+          },
+          handleSubmit,
+          setFieldValue,
+          errors,
+        }): ReactElement => {
+          const modalContact = {
+            id: id,
+            status: statusValue,
+            name: name,
+            pledgeCurrency,
+            pledgeAmount,
+            pledgeFrequency,
+          };
+
+          return (
+            <Form onSubmit={handleSubmit}>
+              <Grid container className={classes.formWrapper}>
+                <Card className={classes.formInner}>
+                  <Grid container>
+                    <Grid item lg={6} md={4} xs={12}>
+                      <Box
+                        display="flex"
+                        p={2}
+                        alignItems="center"
+                        className={classes.left}
+                      >
+                        <Avatar
+                          src=""
+                          style={{
+                            width: theme.spacing(7),
+                            height: theme.spacing(7),
+                          }}
+                        />
+                        <Box display="flex" flexDirection="column" ml={2}>
+                          <Link
+                            data-testid="contactSelect"
+                            underline="hover"
+                            onClick={() => setContactFocus(id, 'Donations')}
+                          >
+                            <Typography variant="h6">{name}</Typography>
+                          </Link>
+                          <Typography>
+                            {`Current: ${statusTitle} ${
+                              amount &&
+                              amountCurrency &&
```

Comment:
For some reason this is showing "0" for partners who do not have an amount defined. It shouldn't show anything.

Reply:
I don't think it matters either way.